srcs/runners_manager/vm_creation/aws/schema.py

```diff
@@ -0,0 +1,8 @@
+from marshmallow import Schema, fields
+
+class AwsConfigVmType(Schema):
+    image_id = fields.Str(required=True)
+    instance_type = fields.Str(required=True)
+    security_group_ids = fields.List(fields.String())
+    subnet_id = fields.Str(required=True)
+    disk_size_gb = fields.Int(required=True)
```

Comment:
I see you removed the main config schema, I don't see the code of the cloudmanager being modified to accept this kind of change (Which surprises me a bit as it should have failed on us).

Please check on the [code](https://github.com/scality/runner-manager/blob/470c4ddc8f0640427c0836af624dee3d0f74d454/srcs/runners_manager/vm_creation/CloudManager.py#L41-L42) if you need to change the condition

Reply:
Indeed it should be modified, but in this case I do not know by what miracle but there was no problem that was trigger

Reply:
I think that `self.CONFIG_SCHEMA` never returned an error because it was never `None` (it is a constant)

Reply:
Yeah I think you're right, anyway, the changes you did are good 👌